### PR TITLE
Cover MessageHelper with UT

### PR DESCRIPTION
**Pull request by indlu**
Cover MessageHelper by unit tests

### CODE REVIEW

**Comment by indlu** [comment]
@LevchenkoS , @okozlovlux, @dcherniev please, review.

File: src/components/application_manager/test/message_helper/message_helper_test.cc [hunk 1]
```diff
@@ -704,135 +779,384 @@ TEST_F(MessageHelperTest, VerifySoftButtonString_CorrectStrings_True) {
 }
 
 TEST_F(MessageHelperTest,
-       GetIVISubscriptionRequests_ValidApplication_HmiRequestNotEmpty) {
```
Comment (LevchenkoS) [inline src/components/application_manager/test/message_helper/message_helper_test.cc]: @indlu , why do you delete this test case?

File: src/components/application_manager/test/message_helper/message_helper_test.cc [hunk 2]
```diff
@@ -238,10 +241,10 @@ TEST(MessageHelperTestCreate,
   DataAccessor<application_manager::CommandsMap> data_accessor(vis, true);
 
   EXPECT_CALL(*appSharedMock, commands_map()).WillOnce(Return(data_accessor));
-  application_manager_test::MockApplicationManager mock_application_manager;
+  application_manager_test::MockApplicationManager mock_application_manager_;
```
Comment (indlu) [inline src/components/application_manager/test/message_helper/message_helper_test.cc]: @LevchenkoS, fixed in d13e7c7

File: src/components/application_manager/test/message_helper/message_helper_test.cc (hunk 1 above)
Comment (indlu) [inline src/components/application_manager/test/message_helper/message_helper_test.cc]: @LevchenkoS, fixed in d13e7c7

**Pull request merged**